packages/stark-ui/src/modules/pagination/components/pagination.component.ts

```diff
@@ -130,15 +130,15 @@ export class StarkPaginationComponent extends MatPaginator implements OnInit, On
 	}
 
 	/**
-	 * Component lifecycle hook
+	 * CodefaultItemsPerPageIsPresentComponent lifecycle hook
```

Comment:
This is incorrect, it should stay "Component lifecycle hook". Could you please adapt this ?